pkg/controller/backup_repository_controller.go

```diff
@@ -53,16 +57,22 @@ const (
 
 type BackupRepoReconciler struct {
 	client.Client
-	namespace            string
-	logger               logrus.FieldLogger
-	clock                clocks.WithTickerAndDelayedExecution
-	maintenanceFrequency time.Duration
-	backupRepoConfig     string
-	repositoryManager    repomanager.Manager
+	namespace                 string
+	logger                    logrus.FieldLogger
+	clock                     clocks.WithTickerAndDelayedExecution
+	maintenanceFrequency      time.Duration
+	backupRepoConfig          string
+	repositoryManager         repomanager.Manager
+	keepLatestMaintenanceJobs int
+	repoMaintenanceConfig     string
+	podResources              kube.PodResources
```

Comment:
trivial:  Can we rename the attribute or add a comment to clarify that this is only for maintenance job?

Reply:
Done, changed it to `maintenanceJobResources`